docs/flip/1009-optimizer-api.md

```diff
@@ -83,9 +83,9 @@ Remarks:
   code illustrates how this can be a drop-in replacement for a function we also
   use in our [previous training step], where this update function already exists
   (notice how we need to invert the sign because we add the gradient update to
-  the parameters). See also [Optax
-  #20](https://github.com/deepmind/optax/issues/20) for an ongoing discussion
-  about scheduling hyper parameters.
+  the parameters). Since
+  [schedule_hyperparams](https://github.com/deepmind/optax/pull/48) Optax
+  supports scheduling arbitrary hyper parameters, not only the learning rate.
```

Comment:
Small suggestion: nit (refactor/grammar)

```suggestion
  the parameters). In addition,
  you can use [`schedule_hyperparams`](https://github.com/deepmind/optax/pull/48) in Optax 
  to schedule the learning rate and other arbitrary hyper parameters.
```

@andsteing LMKWYT

Reply:
Updated similar to your suggestion. Note that the learning rate is schedule differently (using `scale_by_schedule()`) from other hyper parameters.

Reply:
Cool @andsteing Thanks for letting us know